diff --git a/pwa/modules/pwa_a2hs/pwa_a2hs.info.yml b/pwa/modules/pwa_a2hs/pwa_a2hs.info.yml
index e7e14c8..f8934bf 100644
--- a/pwa/modules/pwa_a2hs/pwa_a2hs.info.yml
+++ b/pwa/modules/pwa_a2hs/pwa_a2hs.info.yml
@@ -1,7 +1,7 @@
 name: Progressive Web Add to Home Screen
 description: 'Provides a block with a button to prompt users to add the webapp to their home screen.'
 core: 8.x
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10 || ^11
 type: module
 dependancies:
   - pwa
diff --git a/pwa/modules/pwa_extras/pwa_extras.info.yml b/pwa/modules/pwa_extras/pwa_extras.info.yml
index e1f5353..0399d41 100644
--- a/pwa/modules/pwa_extras/pwa_extras.info.yml
+++ b/pwa/modules/pwa_extras/pwa_extras.info.yml
@@ -1,7 +1,7 @@
 name: Progressive Web App Extras
 description: Builds on top of Progressive Web App to deliver extra functionality.
 core: 8.x
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10 || ^11
 type: module
 configure: pwa_extras.settings
 dependancies:
diff --git a/pwa/pwa.info.yml b/pwa/pwa.info.yml
index ee46919..d48d745 100644
--- a/pwa/pwa.info.yml
+++ b/pwa/pwa.info.yml
@@ -1,7 +1,7 @@
 name: 'Progressive Web App'
 description: 'Make the website installable on mobile.'
 core: 8.x
-core_version_requirement: ^8 || ^9
+core_version_requirement: ^8 || ^9 || ^10 || ^11
 type: module
 configure: pwa.config
 
diff --git a/pwa/src/Controller/PWAController.php b/pwa/src/Controller/PWAController.php
index 3bce084..22d4bf2 100644
--- a/pwa/src/Controller/PWAController.php
+++ b/pwa/src/Controller/PWAController.php
@@ -254,7 +254,7 @@ class PWAController implements ContainerInjectionInterface {
    * @return mixed
    */
   public function pwa_serviceworker_file_data(Request $request) {
-    $path = drupal_get_path('module', 'pwa');
+    $path = \Drupal::service('extension.list.module')->getPath('pwa');
 
     $sw = file_get_contents($path . '/js/serviceworker.js');
 
diff --git a/pwa/src/Form/ManifestConfigurationForm.php b/pwa/src/Form/ManifestConfigurationForm.php
index 08b27ed..944dd1d 100644
--- a/pwa/src/Form/ManifestConfigurationForm.php
+++ b/pwa/src/Form/ManifestConfigurationForm.php
@@ -3,6 +3,7 @@
 namespace Drupal\pwa\Form;
 
 use Drupal\Core\Cache\Cache;
+use Drupal\Core\File\FileUrlGeneratorInterface;
 use Drupal\Core\Cache\CacheTagsInvalidatorInterface;
 use Drupal\Core\Config\ConfigFactoryInterface;
 use Drupal\Core\File\FileSystemInterface;
@@ -155,7 +156,7 @@ class ManifestConfigurationForm extends ConfigFormBase {
   public function buildForm(array $form, FormStateInterface $form_state) {
 
     $host = $this->getRequest()->server->get('HTTP_HOST');
-    $files_path = file_create_url("public://pwa") . '/';
+    $files_path = \Drupal::service('file_url_generator')->generateAbsoluteString("public://pwa") . '/';
     if (substr($files_path, 0, 7) == 'http://') {
       $files_path = str_replace('http://', '', $files_path);
     }
